tests/acceptance/fixtures/html/removablefilters copy.html

```diff
@@ -0,0 +1,143 @@
+<html>
+
+<head>
+  <meta charset="utf-8" />
+
+  <script src="http://localhost:9999/dist/answerstemplates.compiled.min.js"></script>
+  <script src="http://localhost:9999/dist/answers.min.js"></script>
+  <link rel="stylesheet" type="text/css" href="http://localhost:9999/dist/answers.css">
+</head>
+
+<style>
+  .flex {
+    display: flex;
+  }
+</style>
+
+<body>
+  <div class="search-bar"></div>
+  <div class="flex">
+    <div>
+      <div class="facets"></div>
+      <div class="filter-search"></div>
+      <div class="filter-box"></div>
+    </div>
+    <div">
```

Comment:
Extra " here!

Reply:
oh shoot I shouldn't even have this file. Fixed this in the other removablefilters.html though!